includes/themes.php

```diff
@@ -332,7 +334,7 @@ function get_active_theme() {
 	/**
 	 * Retrieves theme information for the current active theme.
 	 *
-	 * @return object
+	 * @return object|string|boolean
```

Comment:
This change seems to be breaking a log of PHPstan rules. I wonder if we need to add a new function without $key so we know an object will always be returned? Something like this?

```
public function get_active_theme_data( $key = null ) {
		$data   = $this->get_active_theme_object();
		if ( ! is_null( $key ) ) {
			return isset( $data->{$key} ) ? $data->{$key} : false;
		}
		return $data;
	}

public function get_active_theme_object() {
		$active = $this->get_active_theme();
		$data   = $this->themes[ $active ];
		return $data;
	}
```